greenplumpython/func.py

```diff
@@ -120,6 +120,18 @@ def create_function(
     return_type_as_name: Optional[str] = None,
     type_is_temp: bool = True,
 ) -> Callable:
+    if not func:
```

Comment:
To be honest, I have spent quite a while to understand why the `func` is `None` here, but did find an answer. Code like this deserves an inline comment.

Reply:
Got it

Reply:
uh, OK! I thought the `None` `func` means the same decorator is applied to the same name functions. :bulb: